modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,178 @@
+
+[id="new-features-cli-0-21-0-release-1-6_{context}"]
+=== CLI 0.21.0 release
+
+* With this update, you can configure a pipeline or task `start` command to return a YAML or JSON-formatted string by using the `--output <string>`, where `<string>` is `yaml` or `json`. Otherwise, without the `--output` option, the `start` command returns a human-friendly message that is hard for other programs to parse. Returning a YAML or JSON-formatted string is useful for continuous integration (CI) environments.  For example, after a resource is created, you can use `yq` or `jq` to parse the YAML or JSON-formatted message about the resource and wait until that resource is terminated without using the `showlog` option. (link:https://github.com/tektoncd/cli/pull/1326[#1326])
+
+* With this update, you can authenticate to a registry with Podman's `auth.json` authentication file. For example, you can use `tkn bundle push` to push to a remote registry using podman instead of docker CLI. (link:https://github.com/tektoncd/cli/pull/1430[#1430])
+
+* With this update, if you use the `tkn taskrun delete --all` command, you can preserve runs that are younger than a specified number of minutes by using the new `--keep-since <minutes>` option. For example, to keep task runs that are less than five minutes old, you enter `tkn taskrun delete -all --keep-since 5`. (link:https://github.com/tektoncd/cli/pull/1435[#1435])
+
+* With this update, when you delete task runs or pipeline runs, you can use the `--parent-resource` and `--keep-since` options together. For example, the `tkn pipelinerun delete --pipeline pipelinename --keep-since 5` command preserves pipeline runs whose parent resource is named `pipelinename` and whose age is five minutes or less. The `tkn tr delete -t <taskname> --keep-since 5` and `tkn tr delete --clustertask <taskname> --keep-since 5` commands work similarly for task runs. (link:https://github.com/tektoncd/cli/pull/1443[#1443])
+
+* This update adds support for the triggers resources to work with `v1beta1` resources (link:https://github.com/tektoncd/cli/pull/1446[#1446], link:https://github.com/tektoncd/cli/pull/1449[#1449], link:https://github.com/tektoncd/cli/pull/1450[#1450], link:https://github.com/tektoncd/cli/pull/1454[#1454], link:https://github.com/tektoncd/cli/pull/1455[#1455])
+
+* This update adds an `ignore-running` option to the `tkn pipelinerun delete` and `tkn taskrun delete` commands. (link:https://github.com/tektoncd/cli/pull/1445[#1445])
+
+* This update adds a `create` subcommand to the `tkn task` and `tkn clustertask` commands. (link:https://github.com/tektoncd/cli/pull/1359[#1359])
+
+[id="new-features-cli-0-20-0-release-1-6_{context}"]
+=== CLI 0.20.0 release
+
+* With this update, if you use the `tkn pipelinerun delete --all` command, you can preserve runs that are younger than a specified number of minutes by using the new `--keep-since <minutes>` option. (link:https://github.com/tektoncd/cli/pull/1392[#1392])
```

Comment:
Basically the same thing is already documented above.

Reply:
Updated throughout. Thanks.